### PR TITLE
docs: use grey logo

### PR DESCRIPTION
**Pull request by necipallef**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,7 @@
 <p align="center">
   <a href="https://fingerprint.com">
     <picture>
```
Comment (JuroUhlar) [inline README.md:3]: Do we still need the picture element?

**Review by JuroUhlar — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (necipallef) [reply README.md:3]: no idea tbh. Let me try without

Comment (necipallef) [reply README.md]: removed it, ty! Could you re-review pls? @JuroUhlar

**Review by JuroUhlar — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (necipallef) [reply README.md]: since there is only 1 image, it made sense to remove it

**Pull request merged**